### DISCUSSION TITLE
more read milvus server without mishard

### DISCUSSION THREAD

**Discussion by TuJianwei** [milvus-io/milvus#4798] [Ideas & Feature requests]
Hello，

By using milvus，I start a write milvus server and a read only milvus server. And I connect writeClient and readLcient separately. And the meta info data shared in mysql,  the /var/lib/milvus/db data shared in cept fs. Could I start 2 or more read milvus server, provide domain name to connet read milver srever without mishard to realize.

Like:  
MilvusClient writeclient = new MilvusClient("ip1", "port1");
MilvusClient readclient = new MilvusClient("read domian", "port");   //this client will distribute 2 or more read milvus server

**Comment by yhmo** [comment]
Theoretically you can manually start multiple read nodes using same mysql and db path.
Make sure the config is correct:
```
cluster:
  enable: true
  role: ro
```

And make sure they have different port.
```
network: 
  bind.address: x.x.x.x
  bind.port: xxxxx
```

**Reply by TuJianwei** [reply]
yeah, I will test it. Because my business is that searching one vector by one requirement, so I need not mishards to disperse the searched vectors and merge then. What I need is that loading many requirements to more read milvus cilent.
Thanks.

**Reply by engrabdu** [reply]
> yeah, I will test it. Because my business is that searching one vector by one requirement, so I need not mishards to disperse the searched vectors and merge then. What I need is that loading many requirements to more read milvus client. Thanks.

how did you connect the mishard with other nodes can you please share your config .because i want to deploy milvus cluster 
thanks

**Reply by yhmo** [reply]
> > yeah, I will test it. Because my business is that searching one vector by one requirement, so I need not mishards to disperse the searched vectors and merge then. What I need is that loading many requirements to more read milvus client. Thanks.
> 
> how did you connect the mishard with other nodes can you please share your config .because i want to deploy milvus cluster thanks

This online document described the steps to deploy mishards:
https://milvus.io/docs/v1.1.1/mishards.md
https://developpaper.com/milvus-practice-distributed-cluster-scheme-based-on-mishards/
